libraries/bezier-rs/src/subpath/solvers.rs

```diff
@@ -123,6 +124,92 @@ impl<ManipulatorGroupId: crate::Identifier> Subpath<ManipulatorGroupId> {
 		// TODO: Consider the shared point between adjacent beziers.
 		inflection_t_values
 	}
+
+	/// Returns the manipulator point that is needed for a miter join if it is possible.
+	pub(crate) fn miter_line_join(&self, other: &Subpath<ManipulatorGroupId>) -> Option<ManipulatorGroup<ManipulatorGroupId>> {
+		let in_segment = self.get_segment(self.len_segments() - 1).unwrap();
+		let out_segment = other.get_segment(0).unwrap();
+		let in_tangent = in_segment.tangent(TValue::Parametric(1.));
+		let out_tangent = out_segment.tangent(TValue::Parametric(0.));
+
+		let normalized_in_tangent = in_tangent.normalize();
+		let normalized_out_tangent = out_tangent.normalize();
+
+		// The tangents must not be parallel for the miter join
```

Comment:
Can you also implement https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-miterlimit please? We need this in Graphite.

Reply:
I'm planning to implement this in a different PR so that we can merge this one first.

I'm thinking of adding this feature via an additional optional param on `offset` and `outline` - does that sound alright to you @Keavon ?